settings.json

```diff
@@ -0,0 +1,600 @@
+            "default": {}
+        },
+        "PREFER_IPV4": {
+            "type": "boolean",
+            "default": false
+        },
+        "RACK_ELEVATION_DEFAULT_UNIT_HEIGHT": {
+            "type": "integer",
+            "default": 22
+        },
+        "RACK_ELEVATION_DEFAULT_UNIT_WIDTH": {
+            "type": "integer",
+            "default": 220
+        },
+        "REDIS_LOCK_TIMEOUT": {
+            "type": "integer",
+            "default": 600
+        },
+        "RELEASE_CHECK_TIMEOUT": {
+            "type": "integer",
+            "default": 86400
+        },
+        "RELEASE_CHECK_URL": {
+            "type": "string",
+            "default": ""
+        },
+        "SANITIZER_PATTERNS": {},
+        "STORAGE_BACKEND": {
+            "type": "string",
+            "default": ""
+        },
+        "STORAGE_CONFIG": {
+            "type": "object",
+            "default": {}
+        },
+        "STRICT_FILTERING": {
+            "type": "boolean",
+            "default": true
+        },
+        "SUPPORT_MESSAHE": {
+            "type": "string",
+            "default": ""
+        },
+        "TEST_FACTORY_SEED": {
+            "type": "string",
+            "default": ""
+        },
+        "TEST_USE_FACTORIES": {
+            "type": "boolean",
+            "default": false
+        },
+        "TEST_PERFORMANCE_BASELINE_FILE": {
+            "type": "string",
+            "default": "nautobot/core/tests/performance_baselines.yml"
+        },
+        "UI_RACK_VIEW_TRUNCATE_FUNCTION": {},
+        "GIT_SSL_NO_VERIFY": {},
+        "NAUTOBOT_LOG_DEPRECATION_WARNINGS": {
+            "type": "boolean",
+            "default": false
+        },
+        "NAUTOBOT_ROOT": {
+            "type": "string",
+            "default": "~/.nautobot/"
+        },
+        "ADMINS": {
+            "type": "array",
+            "default": []
+        },
+        "CSRF_TRUSTED_ORIGINS": {
+            "type": "array",
+            "default": []
+        },
+        "DATE_FORMAT": {
+            "type": "string",
+            "default": "N j, Y"
+        },
+        "SHORT_DATE_FORMAT": {
+            "type": "string",
+            "default": "Y-m-d"
+        },
+        "TIME_FORMAT": {
+            "type": "string",
+            "default": "g:i a"
+        },
+        "DATETIME_FORMAT": {
+            "type": "string",
+            "default": "N j, Y g:i a"
+        },
+        "SHORT_DATETIME_FORMAT": {
+            "type": "string",
+            "default": "Y-m-d H:i"
+        },
+        "DEBUG": {
+            "type": "boolean",
+            "default": false
+        },
+        "FORCE_SCRIPT_NAME": {
+            "type": "string",
+            "default": ""
+        },
+        "INTERNAL_IPS": {
+            "type": "array",
+            "default": [
+                "127.0.0.1",
+                "::1"
+            ]
+        },
+        "LOGGING": {
+            "type": "object",
+            "properties": {
```

Comment:
The LOGGING dict is very configurable so the schema needs to reflect that. At the very least I'd think we need `additioinalProperties: true` at pretty much every level of this object, but it also might be good to move the default values of `formatters`, `handlers`, and `loggers` into just the "default" attribute under each key rather than specifying them as "properties", since `"normal"`, `"verbose"`, `"normal_console"`, `"verbose_console"` are totally arbitrary strings, and under `loggers` potentially any logger name in all of Python/Django/Nautobot/Apps could be used as a key.